<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>427</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>200</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Playlist URL</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="playlistURL"/>
        </item>
        <item>
         <widget class="QPushButton" name="addList">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="prev">
          <property name="minimumSize">
           <size>
            <width>79</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Prev</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="playControl">
          <property name="minimumSize">
           <size>
            <width>79</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Play</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="next">
          <property name="minimumSize">
           <size>
            <width>79</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Next</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="volumeText">
            <property name="minimumSize">
             <size>
              <width>44</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>44</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Volume</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="volumeNumber">
            <property name="minimumSize">
             <size>
              <width>23</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>23</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>100</string>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="volume">
            <property name="maximumSize">
             <size>
              <width>171</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="display">
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideRight</enum>
        </property>
        <widget class="QWidget" name="playLists">
         <attribute name="title">
          <string>Playlist</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <widget class="QListView" name="listView"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="listNumber">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string>0/0</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="log">
         <attribute name="title">
          <string>Log</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QTextBrowser" name="logBrowser"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>volume</sender>
   <signal>valueChanged(int)</signal>
   <receiver>volumeNumber</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
